vllm/platforms/interface.py

```diff
@@ -24,8 +24,8 @@ def is_tpu(self) -> bool:
         return self._enum == PlatformEnum.TPU
 
     @staticmethod
-    def get_device_capability(device_id: int = 0) -> Tuple[int, int]:
-        raise NotImplementedError
+    def get_device_capability(device_id: int = 0) -> Optional[Tuple[int, int]]:
+        return None
```

Comment:
Maybe we should implement a CPU device capability? Curious on your thoughts @youkaichao

Reply:
I would prefer to provide explicit feature checking interfaces, like ```support_compressed_tensor_fp8```, which will make the ```Platform``` more extensible and portable.